docs/quickstart-aks.md

```diff
@@ -0,0 +1,336 @@
+# Quickstart: Open Service Broker for Azure on an Azure Container Service managed cluster
+
+This quickstart walks-through using the Open Service Broker for Azure (OSBA) to
+deploy WordPress on an [Azure Container Service (AKS)](https://azure.microsoft.com/en-us/services/container-service/) managed cluster.
+
+WordPress requires a backend MySQL database. Without OSBA, we would create a database
+in the Azure portal, and then manually configure the connection information. Now
+with OSBA our Kubernetes manifests can provision an Azure Database for MySQL on our behalf,
+save the connection information in Kubernetes secrets, and then bind them to our WordPress instance.
+
+* [Prerequisites](#prerequisites)
+* [Cluster Setup](#cluster-setup)
+  * [Configure your Azure account](#configure-your-azure-account)
+  * [Create a resource group](#create-a-resource-group)
+  * [Create a service principal](#create-a-service-principal)
+  * [Create a Kubernetes cluster using AKS](#create-an-aks-cluster)
+  * [Configure the cluster with Open Service Broker for Azure](#configure-the-cluster-with-open-service-broker-for-azure)
+* [Deploy WordPress](#deploy-wordpress)
+* [Next Steps](#next-steps)
+
+---
+
+## Prerequisites
+
+* A [Microsoft Azure account](https://azure.microsoft.com/en-us/free/).
+* Install the [Azure CLI](#install-the-azure-cli).
+* Install the [Kubernetes CLI](#install-the-kubernetes-cli).
+* Install the [Helm CLI](#install-the-helm-cli).
+
+### Install the Azure CLI
+
+Install `az` by following the instructions for your operating system.
+See the [full installation instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) if yours isn't listed below.
+
+#### MacOS
+
+```console
+brew install azure-cli
+```
+
+#### Windows
+
+Download and run the [Azure CLI Installer (MSI)](https://aka.ms/InstallAzureCliWindows).
+
+#### Ubuntu 64-bit
+
+1. Add the azure-cli repo to your sources:
+    ```console
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | \
+         sudo tee /etc/apt/sources.list.d/azure-cli.list
+    ```
+1. Run the following commands to install the Azure CLI and its dependencies:
+    ```console
+    sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+    sudo apt-get install apt-transport-https
+    sudo apt-get update && sudo apt-get install azure-cli
+    ```
+
+### Install the Kubernetes CLI
+
+Install `kubectl` by running the following command:
+
+```console
+az acs kubernetes install-cli
+```
+
+### Install the Helm CLI
+
+[Helm](https://github.com/kubernetes/helm) is a tool for installing pre-configured applications on Kubernetes.
+Install `helm` by running the following command:
+
+#### MacOS
+
+```console
+brew install kubernetes-helm
+```
+
+#### Windows**
```

Comment:
What's with the `**` in the heading? I don't see a corresponding note anywhere.

Reply:
OOOPS. this was just emphasis before, markdown lint complained because it was treated like a section header. removed.